zone/client_packet.cpp

```diff
@@ -11069,23 +11069,27 @@ void Client::Handle_OP_PickPocket(const EQApplicationPacket *app)
 	//if we do not send this packet the client will lock up and require the player to relog.
 
 	if (victim == this) {
-		Message(0, "You catch yourself red-handed.");
+		Message(Chat::White, "You catch yourself red-handed.");
 	}
 	else if (victim->GetOwnerID()) {
-		Message(0, "You cannot steal from pets!");
+		Message(Chat::White, "You cannot steal from pets!");
```

Comment:
I see these available DB Strings. I assume if its not a valid target, ID 12405 should be used. This can be cleaned up later, don't need to hold this PR up for that.


12405 You must target a player to steal from first.
12406 You must target a player to steal from first.  You may not steal from corpses.
12407 You can't try to steal while STUNNED!
12408 You can't try to steal while attacking.
12409 You can't seem to steal from yourself for some reason...

Reply:
12405 sounds like a client to client PP?

Reply:
It's likely just default failed PP message.

Reply:
Gonna just stick with what I have now, if no objections.